scripts/aws/cfn/microservice.yml

```diff
@@ -467,4 +467,42 @@ Resources:
         WAFName: !Sub '${ProjectName}-delivery-push-io'
         APIGatewayARNs: !GetAtt DeliveryPushMicroservicePublicIoAPI.Outputs.APIGatewayARN
 
-
+  ## Dashboard
+  MandateMicroserviceCloudWatchDashboard:
```

Comment:
@danilosilvase it should be "DeliveryPushMicroserviceCloudWatchDashboard"

Reply:
thanks for review it, updated.